packages/core/test/application_init_spec.ts

```diff
@@ -73,5 +74,55 @@ import {inject, TestBed, waitForAsync} from '../testing';
            });
          })));
     });
+
+    describe('with app initializers represented using observables', () => {
+      let resolve: Subscriber<any>;
```

Comment:
May be rename `resolve` to `subscriber` (so it's more inline with RxJS terminology)?
```suggestion
      let subscriber: Subscriber<any>;
```